Dockerfile-alpine.template

```diff
@@ -4,7 +4,7 @@ ENV NPM_CONFIG_LOGLEVEL info
 ENV NODE_VERSION 0.0.0
 
 RUN addgroup -S node \
-    && adduser -D -S -h /var/cache/node -s /sbin/nologin -G node node \
+    && adduser -D -u 1000 node \
```

Comment:
The node group is added along with the user in a single command.
With `adduser -D -u 1000 node`, you'll get a `node` user with `node` group and UID/GID 1000.